src/js/messaging/containers/Compose.jsx

```diff
@@ -163,6 +166,7 @@ class Compose extends React.Component {
 
 const mapStateToProps = (state) => {
   return {
+    persistFolder: state.folders.data.currentItem.persistFolder,
```

Comment:
Where is this being used? Was it intended for the redirect in `handleConfirmDelete`?